event-sourcing/event-repository/event-repository-jdbc/src/main/java/uk/gov/justice/services/eventsourcing/repository/jdbc/event/EventJdbcRepositoryFactory.java

```diff
@@ -29,4 +31,8 @@ public EventJdbcRepository eventJdbcRepository(final String jndiDatasource) {
                 jndiDatasource,
                 getLogger(EventJdbcRepository.class));
     }
+
+//    public EventJdbcRepository eventJdbcRepository(final DataSource dataSource) throws DatasourceException {
+//        return new EventJdbcRepository(dataSource);
+//    }
```

Comment:
This commented code should be reinstated or removed

Reply:
removed